build/vgg10-radioml/folding_config/ZCU104_folding_config.json

```diff
@@ -1,136 +1,1193 @@
 {
   "Defaults": {},
-  "FMPadding_hls_0": {
-    "SIMD": 2
+  "StreamingFIFO_rtl_0": {
+    "ram_style": "auto",
+    "depth": 339,
+    "impl_style": "rtl",
+    "inFIFODepths": [
+      0
+    ],
+    "outFIFODepths": [
+      0
+    ]
+  },
+  "FMPadding_rtl_0": {
+    "SIMD": 2,
+    "inFIFODepths": [
```

Comment:
The `inFIFODepths` and `outFIFODepths` parameters are set by the automatic FIFO sizing and don't necessarily need to be part of the folding config json. Removing them, might make the folding configuration a little bit shorter and easier to read.